lib/wick-strict-run

```diff
@@ -30,9 +30,16 @@ wickStrictRun() {
     shift
     setOptions=$-
     set +eE
+    # Turning off the error trap for this context.  I don't
```

Comment:
1ed8133f74c89a979e65d402082ea41bf6146f84

I added it as an example.  Would you rather have it in the description?